exporters/stats/prometheus/src/main/java/io/opencensus/exporter/stats/prometheus/PrometheusExportUtils.java

```diff
@@ -177,11 +183,30 @@ public Void apply(CountData arg) {
         new Function<DistributionData, Void>() {
           @Override
           public Void apply(DistributionData arg) {
-            for (long bucketCount : arg.getBucketCounts()) {
+            // For histogram buckets, manually add the bucket boundaries as "le" labels.
+            // See io.prometheus.client.Histogram.java#L329.
+            checkLeLabelInLabelNames(labelNames);
```

Comment:
Shouldn't we validate the views when we first register them with Prometheus, in createDescribableMetricFamilySamples?  If there is an incompatible view, I think it would be better to only log an error once and then skip the view when exporting data.

Reply:
I meant that I think we should only log an error once, in createDescribableMetricFamilySamples.  Then we can slilently skip the incompatible views in createMetricFamilySamples, and avoid creating redundant log statements.

I just noticed that if createDescribableMetricFamilySamples throws an exception, PrometheusStatsCollector.describe skips the remaining views.  Shouldn't it only skip the view that caused an exception to be thrown?

Reply:
Makes sense to me. I added caching to the collector, so that once a `View` is found problematic, the collector should skip its stats in the future.

Reply:
Make sense to me, updated.